cookbook/recipes/development.rb

```diff
@@ -25,3 +25,10 @@
                 "PATH" => "/opt/go/bin:/usr/local/go/bin:#{ENV["PATH"]}"})
   cwd '/home/vagrant/streakers'
 end
+
+execute 'cp -n dotenv.sample .env' do
+  command 'cp -n dotenv.sample .env'
+  environment ({'HOME' => "/home/vagrant",
```

Comment:
yeah. the argument to `execute` will be run if you don' pass  `command` in the block. See if it will work with just the `execute` and the `cwd`.